web/client/components/import/dragZone/enhancers/useFiles.js

```diff
@@ -1,5 +1,29 @@
 
 import { compose, mapPropsStream, withHandlers } from 'recompose';
+import turfBbox from '@turf/bbox';
+import { getConfigProp } from '../../../../utils/ConfigUtils';
+
+
+// retrieves projectionDefs
+const getProjections = () => {
+    const projections = (getConfigProp('projectionDefs') || []).concat([{code: "EPSG:3857", extent: [-20026376.39, -20048966.10, 20026376.39, 20048966.10]},
+        {code: "EPSG:4326", extent: [-180, -90, 180, 90]}
+    ]);
+    return projections;
+};
+
+// finds a projection from projectionDefs
+const getExtentForProjection = code => {
+    return getProjections().find(project => project.code === code) || {extent: [-20026376.39, -20048966.10, 20026376.39, 20048966.10]};
+};
+
+// checks if a layer fits within the max extent
+export const checkIfLayerFitsExtentForProjection = layer => {
+    const crs = layer.bbox?.crs || "EPSG:3857";
+    const { extent } = getExtentForProjection(crs);
+    const [, , maxX, maxY] = turfBbox({type: 'FeatureCollection', features: layer.features || []});
```

Comment:
why you skip minX and minY ?

Reply:
I used the maxX and minY to compare if the max X and Y for the layer can fit within the extent for the projection. Am I doing it wrong @MV88